doc/events.md

```diff
@@ -1,39 +1,38 @@
 # Events
 
-_This functionality is available in versions 0.3.0 and up._
 Note that prior to version 1.0.0 the Google Guava EventBus (`com.google.common.eventbus` package)
 API was used, but now there is an API provided in the `com.cloudant.sync.event` package. To migrate
 your code you will need to adjust your imports accordingly.
 
 Cloudant Sync uses an `EventBus` to post events to interested parties in a publish and subscribe
 fashion.
 
-The `Datastore` class posts events about Documents:
+The `Database` class posts events about Documents:
 
 * `DocumentCreated`
 * `DocumentDeleted`
 * `DocumentUpdated`.
 
-The `DatastoreManager` class posts events about Databases:
+The `DocumentStore` class posts events about DocumentStores:
 
-* `DatabaseClosed`
-* `DatabaseCreated`
-* `DatabaseDeleted`
-* `DatabaseOpened`.
+* `DocumentStoreClosed`
+* `DocumentStoreCreated`
+* `DocumentStoreDeleted`.
```

Comment:
What about `DocumentStoreOpened`? The class still exists and appears to be posted.